vendor/bat-native-ads/src/bat/ads/internal/security/conversions/conversions_util.cc

```diff
@@ -43,7 +44,9 @@ base::Optional<VerifiableConversionEnvelopeInfo> EnvelopeSeal(
     return base::nullopt;
   }
 
-  if (!IsLatinAlphaNumeric(message)) {
+  std::string stripped_message;
+  base::RemoveChars(message, "-", &stripped_message);
```

Comment:
I don't think that's save as `foo-bar` and `foobar` would result in the same ID

Reply:
We only strip for the is Latin check. We still use message thereafter

Reply:
right, could we be more explicit and have a check `IsValidMessage` and match against the actual pattern so we don't have to think about stripping etc.? Probably fine for now though..

Reply:
resolved as per screen share